### DISCUSSION TITLE
DefaultSources.HIKEBIKE_HILLSHADE dead ?

### DISCUSSION THREAD

**Discussion by moiatto** [mapsforge/vtm#1048] [Q&A] [answered]
Seems [https://tiles.wmflabs.org/hillshading](https://tiles.wmflabs.org/hillshading) doesn't work anymore.

is there any other free server?

**Comment by devemux86** [comment, marked as answer]
Hike & Bike is no longer available.

You could try [Esri Hillshade](https://services.arcgisonline.com/arcgis/rest/services/Elevation/World_Hillshade/MapServer), though their tiles are non transparent jpg.

**Comment by moiatto** [comment]
Tried but they are not good unfortunately
